### ISSUE TITLE
Make Floating Action Button WAI-ARIA compliant

### ISSUE THREAD

**Issue by dericgw** [dericgw/react-tiny-fab#1]
I would be nice to have the component accessible for those that use screen readers and such. I am not positive what all goes into making this happen so I am going to start researching it. However, this should be some pretty low hanging fruit.
